README.md

```diff
@@ -42,4 +40,32 @@ The JVM will then listen for a debugger to be attached on port 8189.  The JVM wi
 the debugger to be attached before starting Echo; the relevant JVM arguments can be seen and
 modified as needed in `build.gradle`.
 
-[//]: # "Only here to retrigger the echo build"
+## Configuring
+echo can run in two modes: in-memory and SQL.
+
+
+**In-memory** mode keeps all CRON trigger information in RAM. While this is simpler to configure (this is the default) the in-memory mode doesn't provide for any redundancy because it requires that a single instance of echo scheduler be running. If/when that instance goes down, CRON triggers will not fire.
```

Comment:
I would be extra explicit here: "If there are multiple instances, they will all attempt to start executions for a given CRON trigger. There is no locking, leader election or any other kind of coordination between scheduler instances using the in-memory mode."

Reply:
👍